libvast/src/system/partition_transformer.cpp

```diff
@@ -255,27 +296,40 @@ partition_transformer_actor::behavior_type partition_transformer(
         store_or_fulfill(self, std::move(stream_data));
         return;
       }
-      auto builder_and_header = store_plugin->make_store_builder(
-        self->state.accountant, self->state.fs, self->state.data.id);
-      if (!builder_and_header) {
-        self->state.stream_error
-          = caf::make_error(ec::invalid_argument,
-                            "could not create store builder for backend {}",
-                            store_id);
-        store_or_fulfill(self, std::move(stream_data));
-        return;
-      }
-      self->state.data.store_header = builder_and_header->second;
-      self->state.store_builder = builder_and_header->first;
-      VAST_ASSERT(self->state.store_builder);
       self->state.stage = caf::attach_continuous_stream_stage(
```

Comment:
We can use a (regular?) source instead. Maybe evaluate after the main change is functional.

Reply:
After further discussion, we decided that using a source would be odd since caf assumes that sources are always generators, but we exlusively push to this one so we could do nothing but assert failure in `generate()`.